docs/guides/Getting-Started/getting-started/marketplace-overview.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Marketplace"
+slug: "marketplace-overview"
+hidden: false
+createdAt: "{Creation date as 2022-10-20T17:08:52.219Z}"
+updatedAt: "{Update dates as 2022-10-21T14:33:45.242Z}"
+excerpt: "{Insert a synopsis of your article here}"
+hidePaginationPrevious: false
+hidePaginationNext: false
+---
+# Marketplace
+
+>ℹ️ Help us improve our documentation! Tell us about your experience with this article by filling out [this form](https://forms.gle/fQoELRA1yfKDqmAb8).
+
+We define marketplace as the environment where sellers expose their products and also where consumers make purchases and pay for an order. Sellers in turn are commonly the owners of the products and responsible for delivery the products to consumers. VTEX promotes digital collaboration in different settings. In addition to selling their products in other marketplaces, sellers can become marketplaces themselves. Learn more about partners and growth opportunities in [Marketplace Strategies at VTEX](https://help.vtex.com/en/tutorial/estrategias-de-marketplace-na-vtex--tutorials_402).
```

Comment:
```suggestion
We define marketplace as the environment where sellers expose their products and also where consumers make purchases and pay for an order. Sellers in turn are commonly the owners of the products and responsible for delivering the products to shoppers. Because VTEX promotes digital collaboration in different settings, in addition to selling their products in other marketplaces, sellers can become marketplaces themselves. Learn more about partners and growth opportunities in [Marketplace Strategies at VTEX](https://help.vtex.com/en/tutorial/estrategias-de-marketplace-na-vtex--tutorials_402).
```